script_for_using_sts_assume_role.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# For use this script you need to install jq "sudo apt install jq"
+read -p 'Input token code: ' token_key
+read -p 'Input profile name: ' profile
+json=$(aws sts assume-role --role-arn arn:aws:iam::546240550610:role/Admins \
+--role-session-name tf \
+--serial-number arn:aws:iam::546240550610:mfa/$profile \
```

Comment:
hmm so if i have mfa name like `arn:aws:iam::546240550610:mfa/test` the script will not work right :thinking:

Reply:
Made a serial-number request via aws configure --profile $profile get role_arn